### PR TITLE
Updated ISO certification date

### PR DESCRIPTION
**Pull request by sophia-wiley**
Updated the ISO certification date.

### CODE REVIEW

File: docs/operator-guides/security.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ Airbyte Cloud supports [user management](https://docs.airbyte.com/cloud/managing
 Our compliance efforts for Airbyte Cloud include:
 
 - SOC 2 Type II assessment: An independent third-party completed a SOC2 Type II assessment and found effective operational controls in place. Independent third-party audits will continue at a regular cadence, and the most recent report is available upon request.
-- ISO 27001 certification: We are currently pursuing ISO 27001 certification and will continue to align the evolution of our security program with its standards as we grow.
+- ISO 27001 certification: We received our ISO 27001 certification in November 2022. If you would like a copy of the certification, contact us at security@airbyte.io
```
Comment (sophia-wiley) [inline docs/operator-guides/security.md:150]: @Patsy-Airbyte is security@airbyte.io the email that you would like users to contact if they want a copy of the certification?

Comment (Patsy-Airbyte) [reply docs/operator-guides/security.md:150]: Could we use the same language as the SOC2 section? "A copy of the certificate is available upon request."

Comment (sophia-wiley) [reply docs/operator-guides/security.md]: @Patsy-Airbyte Edited :)

**Review by Patsy-Airbyte — APPROVED** [review]

**Comment by sophia-wiley** [comment]
CC: @Amruta-Ranade

**Pull request merged**